Src/Support/Google.Apis.Auth.AspNetCore/Google.Apis.Auth.AspNetCore.csproj

```diff
@@ -0,0 +1,36 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\CommonProjectProperties.xml" />
+
+  <PropertyGroup>
+    <!-- Only ASP.NET Core 2.0 on netstandard2.0 supported -->
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <!-- Beta release -->
+    <Version>$(Version)-beta01</Version>
+  </PropertyGroup>
+
+  <!-- nupkg information -->
+  <PropertyGroup>
+    <Title>Google APIs ASP.NET Core Auth Extensions</Title>
+    <Description>
+The Google APIs Client Library is a runtime client for working with Google Services.
+
+The ASP.NET Core Auth extension library contains a Google-specific OpenIdConnect auth handler.
+Supports incremental auth, and an injectable IGoogleAuthProvider to supply Google credentials.
+
+Supported Platforms:
+- ASP.NET Core 2.x on netstandard2.x
```

Comment:
I'm not sure that "on netstandard2.x" really makes sense - perhaps "on platforms that support netstandard2.0"? Does ASP.NET Core 2.0 run on anything that doesn't support netstandard2.0 anyway?

Reply:
Good point, removed the part about netstandard as it's unnecessary.